portfolio/src/main/webapp/style.css

```diff
@@ -1,29 +1,108 @@
 #content {
-  font: 'Roboto', sans-serif;
+  font: 22px 'Manrope', sans-serif;
+  line-height: 2;
+  padding: 150px;
   text-align: center;
 }
 
+#intro {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+}
+
+#headshot {
+    border-radius: 100%;
+    padding: 30px;
+    width: 200px;
+}
+
 #name {
-    font: bold 40px 'Inconsolata', monospace;
+    font: bold 80px 'Gochi Hand', cursive;
 }
 
 #about {
-    letter-spacing: 0.5px;
+    align-items: center;
+    display: flex;
+    flex-direction: row;
+    list-style: none;
+    margin: 100px;
+    text-align: left;
 }
 
-#intro {
-    background: rgb(113, 147, 209);
-    color: white;
-    font: 20px;
-    padding: 20px;
-    text-align: center;
+.info-list {
+    display: flex;
+    flex-direction: column;
+    list-style: none;
+    margin: 100px;
+    text-align: left;
 }
 
-#about {
-    padding: 15px 0 0;
+.category {
+    color: rgb(235, 214, 28);
+    letter-spacing: 5px;
+    padding: 50px;
+    text-align: center;
 }
 
 #profiles {
-    font: 20px/10px;
-    padding: 20px 0 0;
+    bottom: 30px;
+    float: right;
+    position: sticky;
+    right: 40px;
+    width: 65px;
+}
+
+#profiles:hover {
```

Comment:
This is super nitpicky so feel free to disagree, but when i mouse over one of the profile links, and all three of the logos change their opacity, it feels to me like they are just one giant button.  It would feel more intuitive (to me) to make the link that is hovered over stand out in some way from the others.

If you wanted to do that, you could create a class called something like profile-link and add that class to each <a> tag in that section.  Then you could add the effect in the CSS by defining properties in .profile-link:hover 

Totally up to you though, so feel free to ignore this =D

Reply:
Second this, but otherwise LG.

Reply:
I agree on this - I'll implement it on a separate branch